### ISSUE TITLE
Parser error for @each $selector in & { ... }

### ISSUE THREAD

**Issue by wesleyvicthor** [leafo/scssphp#438]
Does the `Parser.php` supports this new syntax from `Sass 3.4` ?

I got this error on parsing 
`parse error: failed at `@each $selector-class in & {`

**Comment by Cerdic** [comment]
Since 8ae752575d63b86d963c44e941d6b4b3804eaae5 it's no more failing on parsing, but the compilation result is wrong:

```
.foo, .bar, .foo2 .bar2, .foo3 > .bar3 {

	@each $selector in & {
		content: "#{$selector}";
	}
}
```
gives

```
.foo, .bar, .foo2 .bar2, .foo3 > .bar3 {
  content: ".foo, .bar, .foo2.bar2, .foo3>.bar3"; }
```

see also #310

**Comment by Cerdic** [comment]
Fix with #693
